plugins/node/opentelemetry-instrumentation-mysql/src/instrumentation.ts

```diff
@@ -32,10 +32,11 @@ import {
   DbSystemValues,
   SemanticAttributes,
 } from '@opentelemetry/semantic-conventions';
-import type * as mysqlTypes from 'mysql';
+import * as mysqlTypes from 'mysql';
 import { MySQLInstrumentationConfig } from './types';
 import { getConnectionAttributes, getDbStatement, getSpanName } from './utils';
 import { VERSION } from './version';
+import { UpDownCounter, MeterProvider } from '@opentelemetry/api-metrics';
```

Comment:
```suggestion
import { UpDownCounter, MeterProvider } from '@opentelemetry/api';
```

NOTE: For this suggestion, we'd also need to remove `@opentelemetry/api-metrics` from the dependencies `package.json` :slightly_smiling_face:

Reply:
done